heaps/heap_sort.py

```diff
@@ -1,8 +1,13 @@
 
 
+from heaps.min_heap import MinHeap
+
+
 def heap_sort(list):
```

Comment:
✨ Very nice, You are creating another list which you are returning so space complexity here would be O(n)